CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 Changelog
 =========
-
+- Adds second derivatives of contravariant basis vectors to the data index.
```

Comment:
I think "list of things we can compute" or something like that is better than "data index" since that's more internal to the code.